src/main/java/calculator/ArithmeticCalculator.java

```diff
@@ -0,0 +1,31 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.function.BinaryOperator;
+import type.MessageFactory;
+
+enum ArithmeticCalculator {
+    PLUS("+", (e1, e2) -> e1 + e2),
+    MINUS("-", (e1, e2) -> e1 - e2),
+    DIVISION("/", (e1, e2) -> e1 / e2),
+    MULTI("*", (e1, e2) -> e1 * e2);
+
+    private String operator;
+    private BinaryOperator<Double> binaryOperator;
```

Comment:
```suggestion
    private final String operator;
    private final BinaryOperator<Double> binaryOperator;
```

enum의 경우 필드에 final을 붙여주는 것이 관례인데 어떠한 이유일지 고민해보아요

Reply:
아 몰랐었는데 감사합니다 👍